easybuild/easyconfigs/p/PGI/PGI-16.3-GCC-4.9.3-2.25.eb

```diff
@@ -0,0 +1,25 @@
+name = 'PGI'
+version = '16.3'
+
+homepage = 'http://www.pgroup.com/'
+description = "C, C++ and Fortran compilers from The Portland Group - PGI"
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['pgilinux-20%(version_major)s-%(version_major)s%(version_minor)s-x86_64.tar.gz']
+checksums = ['618cb7ddbc57d4e4ed1f21a0ab25f427']
+
+gccver = '4.9.3'
+binutilsver = '2.25'
+versionsuffix = '-GCC-%s-%s' % (gccver, binutilsver)
+
+dependencies = [
+    ('GCCcore', gccver),
+    ('binutils', binutilsver, '', ('GCCcore', gccver)),
+]
+
+# license file
+import os
+license_file = os.path.join(os.getenv('HOME'), "licenses", "pgi", "license.dat")
```

Comment:
@bartoldeman please use the (new since EB 2.7.0) `HOME` constant here, rather than importing `os`:

```python
license_file = HOME + '/licenses/pgi/license.dat'
```

otherwise: good to go